### ISSUE TITLE
Update contribution guidelines

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#2341]
### Please describe the problem (or idea)
At public labs we are now using a new pr label and issue label system as discussed in #2231. One problem with the new label system is the newcomers are not having knowledge of our existing label system
> What happened just before the problem occurred? Or what problem could this idea solve?
I think one solution is to include the link to the contributors page (https://github.com/publiclab/plots2/blob/master/CONTRIBUTING.md) in all the first timers issue so that they will read and do the prs accordingly.
For that we need to change the contributors page 
This is a part of #2231 
@jywarren @ebarry what do you suggest??

**Comment by jywarren** [comment]
Agreed -- this sounds great. Also we can change
https://github.com/publiclab/plots2/blob/master/.github/PULL_REQUEST_TEMPLATE.md
or https://github.com/publiclab/plots2/blob/master/.github/ISSUE_TEMPLATE.md
as well. I'd be happy to review a PR about this with an initial copy of
suggested labels maybe from #2231, but are we sure non-contributors can add
labels themselves?

**Comment by SidharthBansal** [comment]
I am little bit inclined to teaching them how to add labels if they don't know how to add them.
It is easy. Only they need to select labels from sidebar

**Comment by jywarren** [comment]
Oh, sorry i meant like are we sure they even have permissions to change labels? I sometimes forget what i'm able to do as a maintainer :-)

**Comment by jywarren** [comment]
+1 teaching!

**Comment by SidharthBansal** [comment]
No, they can't add labels. So, let's think what alternative can be adopted.

**Comment by SidharthBansal** [comment]
My mistake 😢

**Comment by jywarren** [comment]
if a label is assigned to them, can they then add labels?

**Comment by jywarren** [comment]
np! There might be something we could do with a bot, maybe, but I don't
know. We could ask them to ping someone for labeling help via
`@publiclab/______`?

**Comment by SidharthBansal** [comment]
We can check this. I will create a new account now and see what happens

**Comment by bansalsidharthcode** [comment]
Asking them to ping is a great option
Can you please assign this issue to this id so that I can check if anyhow I can add labels to this specific issue.

**Comment by jywarren** [comment]
oh, darn -- you can't assign issues to someone who's not a contributor! :-(

**Comment by SidharthBansal** [comment]
This task is published on GCI dashboard. Thanks all.

**Comment by oorjitchowdhary** [comment]
Is this issue still open? I'd like to work on it... I'm a GCI candidate.

**Comment by SidharthBansal** [comment]
Yeah

**Comment by oorjitchowdhary** [comment]
https://github.com/publiclab/plots2/blob/master/CONTRIBUTING.md
I have to update this right?
And what all is the necessary and relevant info that could be added here...

**Comment by SidharthBansal** [comment]
#2231

**Comment by SidharthBansal** [comment]
Location is right

**Comment by oorjitchowdhary** [comment]
Which is the file to be updated? Or I have to make a PR for this issue?

**Comment by SidharthBansal** [comment]
I think we need to create a new file in doc folder. 
Can you define all the labels there?
We will review and tell about the patches?
You can refer other documents to see how to write the documents.

**Comment by oorjitchowdhary** [comment]
Ok... It can be like this file right? https://github.com/publiclab/plots2/blob/master/doc/GCI_MENTOR_GUIDELINES.md

**Comment by SidharthBansal** [comment]
Yeah

**Comment by oorjitchowdhary** [comment]
Ohk.. Then I'll claim this task on the GCI platform

**Comment by oorjitchowdhary** [comment]
Pls review #3981 ... Thanks.. 😄

**Comment by SidharthBansal** [comment]
@oorjitchowdhary can you please close #2341 and #2231 once #3981 is merged?

**Comment by oorjitchowdhary** [comment]
@SidharthBansal Okay sure
